client/src/App.js

```diff
@@ -90,38 +91,39 @@ class App extends Component {
             <Router basepath={process.env.PUBLIC_URL} className="app" >
               <LandingPageBody path="/" />
               <SiteMap path="/SiteMap" />
+              <Profile path="/Profile" user={state.main.user}/>
               <Error actions={actions} default />
-              <AboutLab1 path="/Lab1/"/>
-              <AboutLab2 path="/Lab2/"/>
-              <AboutLab3 path="/Lab3/"/>
-              <AboutLab4 path="/Lab4/"/>
-              <AboutLab5 path="/Lab5/"/>
-              <AboutLab1 path="/Lab1/About"/>
-              <AboutLab2 path="/Lab2/About"/>
-              <AboutLab3 path="/Lab3/About"/>
-              <AboutLab4 path="/Lab4/About"/>
-              <AboutLab5 path="/Lab5/About"/>
+              <AboutLab1 path="/Lab1/" user={state.main.user}/>
```

Comment:
Do you want to maybe destructure the {state.main.user} to maybe const { user } = state.main
This doesn't have to be now so it doesn't delay the roll out.